sentry_sdk/integrations/logging.py

```diff
@@ -248,27 +249,26 @@ def _emit(self, record):
             event["level"] = level  # type: ignore[typeddict-item]
         event["logger"] = record.name
 
-        # Log records from `warnings` module as separate issues
-        record_captured_from_warnings_module = (
-            record.name == "py.warnings" and record.msg == "%s"
-        )
-        if record_captured_from_warnings_module:
-            # use the actual message and not "%s" as the message
-            # this prevents grouping all warnings under one "%s" issue
-            msg = record.args[0]  # type: ignore
-
-            event["logentry"] = {
-                "message": msg,
-                "formatted": record.getMessage(),
-                "params": (),
-            }
-
+        if (
+            sys.version_info < (3, 11)
```

Comment:
Note: this additional version check does not change actual behavior. In Python 3.11+, `record.msg` contains the actual warning message, so the later on `record.msg == "%s"` check would fail. Adding this version check just makes it more obvious that this code path only exists for older Python versions